pan3d/utils.py

```diff
@@ -51,9 +51,10 @@ async def coroutine():
     singleton_task = asyncio.run_coroutine_threadsafe(coroutine(), task_loop)
 
 
-def is_cloud_env():
-    cloud_env_vars = ["BINDER_REQUEST"]
-    return any(os.environ.get(k) for k in cloud_env_vars)
+def force_local_rendering():
```

Comment:
This method does not `force_local_rendering`. If I'm not mistaken, the intent of that method it to check if you have a GPU which then get used for something else such as "should I force client side rendering".

Reply:
I see the distinction you mean. How about `has_gpu_rendering`?

Reply:
Change made in [094c3bc](https://github.com/Kitware/pan3d/pull/48/commits/094c3bc80ec74777fbd3ec35a92a6299f575a126)